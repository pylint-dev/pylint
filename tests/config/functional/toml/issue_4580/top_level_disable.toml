[tool.pylint]
# disable does not belong in top level section
disable = "logging-not-lazy,logging-format-interpolation"
