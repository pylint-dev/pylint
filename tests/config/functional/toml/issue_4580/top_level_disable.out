************* Module {abspath}
{relpath}:1:0: E0014: Incorrect setting encountered in top level configuration-section 'disable' : 'logging-not-lazy,logging-format-interpolation' (bad-configuration-option-value)
